docs/0-installation.md

```diff
@@ -112,6 +112,49 @@ Draper::CollectionDecorator.send :delegate, :per_page_kaminari
 
 If you're getting the error `wrong number of arguments (6 for 4..5)`, [read #2703].
 
+## webpacker
+
+Since rails 6.0 webpacker has become the default asset generator. You can opt in using webpacker for ActiveAdmin assets as well by setting `use_webpacker` at install or manually.
+
+* at active_admin installation:
+
+  ```sh
+  rails g active_admin:install --use_webpacker
+  ```
+
+* manually:
+
+  ```ruby
+  ActiveAdmin.setup do |config|
+    config.use_webpacker = true
+  end
+  ```
+
+  And run the generator to get default Active Admin assets:
+
+  ```sh
+  rails g active_admin:webpacker
+  ```
+
+You will need to add `@activeadmin/activeadmin` package to your `package.json`:
+
+```sh
+yarn add @activeadmin/activeadmin
+```
+
+Either way, Active Admin needs jquery to be available through webpack plugin, so if you haven't already, add this to your `config/webpack/environment.js`:
+
+```javascript
+const webpack = require('webpack')
+environment.plugins.prepend('Provide',
+  new webpack.ProvidePlugin({
+    "$":"jquery",
+    "jQuery":"jquery",
+    "window.jQuery":"jquery"
+  })
+)
+```
```

Comment:
Since we have a webpacker generator now (thanks!) could we not include this there? There is an option to insert into an existing file or append to a file so this should be easy to automate. I'd suggest going off [some of the existing install files in webpacker](https://github.com/rails/webpacker/tree/master/lib/install) to get ideas. With that change then you'd also be able to remove this code from the support template too.

Reply:
Good point! PR updated accordingly